README.md

```diff
@@ -14,48 +14,49 @@ Headless Chat SDK to build your own chat widget against Dynamics 365 Omnichannel
 Please make sure you have a chat widget configured before using this package or you can follow this [link](https://docs.microsoft.com/en-us/dynamics365/customer-service/add-chat-widget)
 
 ## Table of Contents
+
 - [Live Chat Widget vs. Chat SDK](#live-chat-widget-vs-chat-sdk)
 - [Releases](#releases)
 - [Installation](#installation)
 - [Installation on React Native](#installation-on-react-native)
 - [SDK Methods](#sdk-methods)
-    - [Initialization](#initialization)
-    - [Start Chat](#start-chat)
-    - [End Chat](#end-chat)
-    - [Get Pre-Chat Survey](#get-pre-chat-survey)
-    - [Get Live Chat Config](#get-live-chat-config)
-    - [Get Current Live Chat Context](#get-current-live-chat-context)
-    - [Get Data Masking Rules](#get-data-masking-rules)
-    - [Get Chat Reconnect Context](#get-chat-reconnect-context)
-    - [Get Conversation Details](#get-conversation-details)
-    - [Get Chat Token](#get-chat-token)
-    - [Get Calling Token](#get-calling-token)
-    - [Get Messages](#get-messages)
-    - [Send Messages](#send-messages)
-    - [On New Message](#on-new-message)
-    - [On Typing Event](#on-typing-event)
-    - [On Agent End Session](#on-agent-end-session)
-    - [Send Typing Event](#send-typing-event)
-    - [Email Live Chat Transcript](#email-live-chat-transcript)
-    - [Get Live Chat Transcript](#get-live-chat-transcript)
-    - [Upload File Attachment](#upload-file-attachment)
-    - [Download File Attachment](#download-file-attachment)
-    - [Create Chat Adapter](#create-chat-adapter)
-    - [Get Voice & Video Calling](#get-voice--video-calling)
-    - [Get Post Chat Survey Context](#get-post-chat-survey-context)
+  - [Initialization](#initialization)
+  - [Start Chat](#start-chat)
+  - [End Chat](#end-chat)
+  - [Get Pre-Chat Survey](#get-pre-chat-survey)
+  - [Get Live Chat Config](#get-live-chat-config)
+  - [Get Current Live Chat Context](#get-current-live-chat-context)
+  - [Get Data Masking Rules](#get-data-masking-rules)
+  - [Get Chat Reconnect Context](#get-chat-reconnect-context)
+  - [Get Conversation Details](#get-conversation-details)
+  - [Get Chat Token](#get-chat-token)
+  - [Get Calling Token](#get-calling-token)
+  - [Get Messages](#get-messages)
+  - [Send Messages](#send-messages)
+  - [On New Message](#on-new-message)
+  - [On Typing Event](#on-typing-event)
+  - [On Agent End Session](#on-agent-end-session)
+  - [Send Typing Event](#send-typing-event)
+  - [Email Live Chat Transcript](#email-live-chat-transcript)
+  - [Get Live Chat Transcript](#get-live-chat-transcript)
+  - [Upload File Attachment](#upload-file-attachment)
+  - [Download File Attachment](#download-file-attachment)
+  - [Create Chat Adapter](#create-chat-adapter)
+  - [Get Voice & Video Calling](#get-voice--video-calling)
+  - [Get Post Chat Survey Context](#get-post-chat-survey-context)
 - [Common Scenarios](#common-scenarios)
-    - [Using BotFramework-WebChat](#using-botframework-webchat)
```

Comment:
Is the list still under “Common Scenarios”?

Reply:
yes